pcdsdevices/mpod.py

```diff
@@ -181,3 +181,36 @@ def __init__(self, channel_prefix, card_prefix, name='mpod_hv_channel',
                  **kwargs):
         self._card_prefix = card_prefix
         super().__init__(channel_prefix, name=name, **kwargs)
+
+
+def MPOD(channel_prefix, card_prefix=None, **kwargs):
+    """
+    Determine the appropriate MPOD Channel class based on the channel number.
+
+    Parameters
+    ----------
+    channel_prefix : str
+        Epics base PV for channels.
+    card_prefix : str
+        Epics base PV for the whole MPOD module, HV Channels.
+    kwargs : dict
+        Information to pass through to the device, upon initialization
+    """
+    # CH below 100 should be LV, and above or 100 should be HV.
```

Comment:
This isn't true, it just happens to be true in our example. The real pattern is:
- Channels 0-7 are on the first card
- Channels 100-107 are on the second card
- Channels 200-207 are on the third card

...etc.

And any card can be HV or LV.

Reply:
Ohh noo... thank you for catching this @ZLLentz  - do you by any chance know how we can recognize if it is a HV or LV from the questionnaire run details? 
I'm going to try to look for more examples....

Reply:
As of right now, it's pretty much the voltage range. You might be able to go from that but if you think it's too flimsy, a can be introduced that all HV cards must say 'HV' in their purpose field or something.

Reply:
Being able to get the info directly from the questionnaire purpose field would be great, how do we go about that exactly?

Reply:
I actually do not think it s a good idea to use the purpose for this. I do not want to make filling this out hard. The purpose is there so that they can make sure they have thought of all their needs and that is it.  We have the user put in the max voltages so that we can decide which channel to use and populate in the questionnaire. The software should then figure out if this is a high or low voltage module, either by checking on presence of the high/rise setting per channel or, somewhat prettier, using the results of the config max V w/ using e.g. 50V as line between (I think I have a 30V LV module and a 500V HV module, so in between.

Reply:
Ok thanks for letting me know @silkenelson. I will try to use the `GetMaxVoltage` somehow to determine that instead

Reply:
Are these reconfigured frequently? 

Could we just pre-build a list of channels, their card prefix, and whether they are HV/LV?

Reply:
I was writing my other question here before you asked this one @klauer, - my question was:
And just to make sure, I need to make up the card_prefix PV, am I safe to assume: 
ch: 100-107 : card_prefix = ...:MPD:MOD:10
ch: 200-207 : card_prefix = ...:MPD:MOD:20
and so on... what does the `card_prefix` for the first card looks like...

But what do you mean pre-build a list of channels? kind of what we have for Acromag, all the channels in the object? but here all the channels on the card?

Reply:
They do not change particular often (actually rather rarely). Each Mpod has its own order&type of cards though, they are clearly not created all equal. Where would we store this information?

Reply:
Would it be wrong to create full MPOD objects and store them in happi?

Reply:
I think this would be a reasonable thing to do.

Reply:
@ZLLentz can you please elaborate on creating full MPOD objects and store them in happi?
Where can I get the info of all possible MPOD objects' order&type of cards?

Reply:
Honestly it ends up being a bit of a chore. We can harvest `/cds/group/pcds/epics/ioc/common/mpod/R2.2.3/iocBoot` for the configurations, then we need classes for each MPOD configuration (or at least for each low voltage and each high voltage card?). Then we have to to add all the instances to `happi`... and then in the questionnaire load we have to figure out which MPOD card a channel belongs to by context clues in the PV. It's quite a bit of overhead when you think it all the way through... I've kind of soured on the idea as a solution for the questionnaire issue.

Reply:
So out of all the iocs in `/cds/group/pcds/epics/ioc/common/mpod/R2.2.3/iocBoot` I'm able to get the Pvs information for 7 of them, the others say `Not Found` 
Each MPOD configuration has at least one High Voltage card - I was able to get the PVs for the channels but not for the whole cards.
Should I add MPOD classes for the ioc configurations that I was able to get PVs data from or for all the ioc configurations from that directory?